src/op.cpp

```diff
@@ -2231,9 +2231,8 @@ bool jitc_can_scatter_reduce(JitBackend backend, VarType vt, ReduceOp op) {
             if (op == ReduceOp::Add && compute_capability < 60)
                 return false;
 
-            // Half-precision min/max reductions require sm_90
-            if ((op == ReduceOp::Min || op == ReduceOp::Max) &&
-                compute_capability < 90)
+            // Half-precision reductions other than `add` are not supported on CUDA
+            if (is_cuda && op != ReduceOp::Add)
```

Comment:
I have left the implementation in `cuda_scatter.cpp` intact.
Ideally, we would be able to still support those reductions with a workaround.

Reply:
Actually I think changing this function is not enough to clearly indicate to the user that this reduction is not supported.